src/State/Provider/DeserializeProvider.php

```diff
@@ -114,4 +117,30 @@ public function provide(Operation $operation, array $uriVariables = [], array $c
 
         return $data;
     }
+
+    private function normalizeExpectedTypes(?array $expectedTypes = null): array
+    {
+        $normalizedTypes = [];
+
+        foreach ($expectedTypes ?? [] as $expectedType) {
+            $normalizedType = $expectedType;
+
+            if (class_exists($expectedType) || interface_exists($expectedType)) {
+                try {
+                    $normalizedType = $this->resourceMetadataCollectionFactory->create($expectedType)->getOperation()->getShortName();
```

Comment:
```suggestion
                    $normalizedType = $this->resourceMetadataCollectionFactory->create($expectedType)->getOperation()?->getShortName();
```

Reply:
I have checked `\ApiPlatform\Metadata\Resource\ResourceMetadataCollection::getOperation`, it always returns `Operation`.